packages/main/src/ComboBox.hbs

```diff
@@ -27,6 +27,12 @@
 		data-sap-focus-ref
 	/>
 
+	{{#if _effectiveShowClearIcon}}
+		<div @click={{_clear}} class="ui5-input-clear-icon-wrapper" input-icon tabindex="-1">
+			<ui5-icon tabindex="-1" class="ui5-input-clear-icon" name="decline" accessible-name="{{clearIconAccessibleName}}"></ui5-icon>
```

Comment:
for now, it will remain an img with no aria-hidden, as the icon does not provide an API which manipulates the aria-hidden attribute in the shadow dom. It is known and will be discussed with other ACC expert how we could handle it.

Reply:
there is aria-hidden property which accepts some string, so basically providing a truthy value should set the aria-hidden attribute

Reply:
Discussed offline, so resolving the conversation